rel/reltool.config.script

```diff
@@ -36,6 +36,7 @@ BaseAppsToInclude = AppsToRun ++
                      ranch,
                      cowlib,
                      bear,
+                     jsx,
                      mochijson2,
```

Comment:
I guess, It has to be removed from here too. Travis build is failing, probably this is the reason.